### PR TITLE
Add SECURITY.md with email for contact.

### LINKED ISSUE #4797: Trying to get in touch regarding a security issue
Hi there,

I couldn't find a `SECURITY.md` in your repository and am not sure how to best contact you privately to disclose a security issue.

Can you add a `SECURITY.md` file with an e-mail to your repository, so that our system can send you the vulnerability details? GitHub suggests that a [security policy](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) is the best way to make sure security issues are responsibly disclosed.

Once you've done that, you should receive an e-mail within the next hour with more info.

Thanks! (cc @huntr-helper)

### PR DESCRIPTION
**Pull request by sculpt0r**
## What is the purpose of this pull request?

Add security contact file.

## Does your PR contain necessary tests?

All patches that change the editor code must include tests. You can always read more
on [PR testing](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_contributing_code.html#tests),
[how to set the testing environment](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html) and
[how to create tests](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_tests.html#creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

## Did you follow the CKEditor 4 code style guide?

Your code should follow the guidelines from the [CKEditor 4 code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md) which helps keep the entire codebase consistent.

- [X] PR is consistent with the code style guide

## What is the proposed changelog entry for this pull request?

```
skip
```

## What changes did you make?

Used our SECURITY file from CKE5.

## Which issues does your PR resolve?

Closes #4797

### CODE REVIEW

**Comment by f1ames** [comment]
Should go to default branch which is `major`.

**Comment by f1ames** [comment]
Rebased onto latest `major`.

**Review by f1ames — APPROVED** [review]
LGTM :+1:

**Comment by f1ames** [comment]
@godai78 could you take a final look on this one?

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Reporting a security issues
+
+If you believe you have found a security issue in the CKEditor 4 software, please contact us immediately.
+
+When reporting a potential security problem, please bear this in mind:
+
+*   Make sure to provide as many details as possible about the vulnerability.
+*   Please do not disclose publicly any security issues until we fix them and publish security releases.
+
+Contact the security team at security@cksource.com. As soon as we receive the security report, we'll work promptly to confirm the issue and then to provide a security fix.
```
Comment (godai78) [inline SECURITY.md:10]:
```suggestion
Contact the security team at security@cksource.com. As soon as we receive the security report, we will work promptly to confirm the issue and then to provide a security fix.
```

**Review by godai78 — APPROVED** [review]
A mannerism more than a correction ;)

**Pull request merged**